src/LibraryManager/Providers/Cdnjs/CdnjsProvider.cs

```diff
@@ -97,6 +97,11 @@ public ILibraryCatalog GetCatalog()
                         return LibraryInstallationResult.FromCancelled(desiredState);
                     }
 
+                    if (string.IsNullOrEmpty(file))
+                    {
+                        return new LibraryInstallationResult(desiredState, PredefinedErrors.CouldNotWriteFile(file));
```

Comment:
Would this error look weird when there is no file name? Perhaps a different error would be better - like "Could not write file because file name is missing" or something like that?

Reply:
The error is: {0} could not be written to disk. Make sure the file name is correct. I could change this to: File "{0}" could not be written to disk. Make sure the file name is correct.

Reply:
This will require new loc changes at this point. I can log a bug for post preview to get these fixed.

Reply:
I think the existing message seems fine.

Reply:
Logged a bug to review error message. Will push changes as is for preview4.